### PR TITLE
extend readme and describe how to read tests

### PR DESCRIPTION
**Pull request by fschade**
add section to readme that describes how we run tests, what to note and how to read results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
 $ ./scripts/cdperf --cloud-docker-host=ssh://user@your-host --cloud-vendor=ocis --k6-test-host=https://your-host:9200
 ```
 
+## How to test
+It's important to know how to compare the tests against each other and what those numbers mean.
+
+**Please note the following points:**
+* Only compare clouds if they run on the same host
+* Try to run the cloud on a different host then k6
```
Comment (phil-davis) [inline README.md:42]:
```suggestion
* Try to run the cloud on a different host than the k6 test-runner
```

File: README.md [hunk 2]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
...
+* Try to keep traffic on the hosts as low as possible while testing
+* Don't compare clouds that run in docker against non dockerized clouds
+* Docker for macs is slow on file operations compared to linux
+* Sometimes it's possible that one of the clouds will fail on some operations. Keep in mind that it's not possible to compared a test with failures against a test where all checks are green
```
Comment (phil-davis) [inline README.md:46]:
```suggestion
* Sometimes it's possible that one of the clouds will fail on some operations. Keep in mind that it's not valid to compare a test with failures against a test where all checks are green
```

File: README.md [hunk 3]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
...
+
+**Test setup at ownCloud:**
+
+At ownCloud we currently test two times a day on different servers. Server (A) is intel based and server (B) amd based.
+The first test run, runs the tests on A which is testing on B and then B which is testing on A.
```
Comment (phil-davis) [inline README.md:51]:
```suggestion
The first test run, runs the tests on A which is testing a server on B and then B which is testing on A.
```

File: README.md [hunk 4]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
...
+We collect those metrics over time to get indicators how the performance changes over time (version to version) and how the clouds perform in comparison to each other.
```
Comment (phil-davis) [inline README.md:52]:
```suggestion
We collect those metrics over time to get indicators of how the performance changes over time (version to version) and how the clouds perform in comparison to each other.
```

File: README.md [hunk 5]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
...
+
+**How to read the test results**
+
+Let's use 'test-issue-github-ocis-1018-propfind-flat.js' as example.
```
Comment (phil-davis) [inline README.md:56]:
```suggestion
Let's use 'test-issue-github-ocis-1018-propfind-flat.js' as an example.
```

File: README.md [hunk 6]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
...
+* Total time of execution
+    * This is the time how long the test took for all users and iterations
```
Comment (phil-davis) [inline README.md:58]:
```suggestion
    * This is the total elapsed time of the test for all users and iterations
```

File: README.md [hunk 7]
```diff
@@ -34,6 +34,38 @@ $ # with cloud on remote docker host
...
+* status
+    * Red || Green is a quick overview how many requests are failed or not
```
Comment (phil-davis) [inline README.md:60]:
```suggestion
    * Red || Green is a quick overview of how many requests failed or not
```

**Comment by fschade** [comment]
thanks @phil-davis , updated 👍

**Review by phil-davis — APPROVED** [review]

**Pull request merged**